src/components/Table/components/TableHeader.scss

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap');
+
+@import '../../../utilis/colors';
+@import '../../../utilis/mixins';
+
+.table-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 1rem;
+  height: 70px;
+
+  .header-content {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding: 1.4375rem 0;
+
+    .header-title {
+      @include gilroy-medium500();
+      font-size: 1.2rem;
+      color: $text-color-dark-gray;
+    }
+
+    .project-count {
+      @include gilroy-medium500();
+      display: flex;
+      height: 1.875rem;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      border-radius: 0.25rem;
+      padding: 0.625rem;
```

Comment:
is this 10px?. This is so hard to read.

Reply:
And also I think this is not to much important to be in relative units. We can fix this to suite design